### ISSUE TITLE
[Bug] Failed to load dynamic library 'libdart_ffi.so

### ISSUE THREAD

**Issue by AmirRezaM75** [AppFlowy-IO/AppFlowy#3240]
### Bug Description

It throws error when opening the app.

```
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: Invalid argument(s): Failed to load dynamic library 'libdart_ffi.so': libssl.so.1.1: cannot open shared object file: No such file or directory
#0      _open (dart:ffi-patch/ffi_dynamic_library_patch.dart:11)
#1      new DynamicLibrary.open (dart:ffi-patch/ffi_dynamic_library_patch.dart:22)
#2      _open (package:appflowy_backend/ffi.dart:27)
#3      _dart_ffi_lib (package:appflowy_backend/ffi.dart:10)
#4      _invoke_log (package:appflowy_backend/ffi.dart)
#5      log (package:appflowy_backend/ffi.dart)
#6      Log.error (package:appflowy_backend/log.dart:61)
#7      PlatformErrorCatcherTask.initialize.<anonymous closure> (package:appflowy/startup/tasks/platform_error_catcher.dart:16)
#8      PlatformDispatcher._dispatchError (dart:ui/platform_dispatcher.dart:1195)
#9      _onError (dart:ui/hooks.dart:117)
```

### How to Reproduce
Install libc6 as mentioned here
https://github.com/Smartmind12/AppFlowy-Docs/blob/main/essential-documentation/install-appflowy/installation-methods/troubleshooting-when-installing-appflowy.md
and https://github.com/AppFlowy-IO/AppFlowy/issues/180

```
sudo apt upgrade libc6
```

Download [AppFlowy_0.2.9_ubuntu-20.04.deb](https://github.com/AppFlowy-IO/AppFlowy/releases/download/0.2.9/AppFlowy_0.2.9_ubuntu-20.04.deb).
Install the app with following command:
```
 sudo dpkg -i AppFlowy_0.2.9_ubuntu-20.04.deb
```

Run the app in terminal

```bash
AppFlowy
```

### Expected Behavior

Run the app normally without facing blank window.

### Operating System

Ubuntu 22

### AppFlowy Version(s)

0.2.9

**Comment by annieappflowy** [comment]
Have you checked [this](https://app.gitbook.com/o/IHfa17qNq2j7xRHMJFoU/s/vs4LQcuzr0JR34ApS5sM/essential-documentation/contribute-to-appflowy/software-contributions/troubleshooting) FAQ?

**Comment by rileyhawk1417** [comment]
@annieappflowy the link you posted leads to a login page for gitbook

**Comment by AmirRezaM75** [comment]
@annieappflowy The gitbook service is not available in my country

**Comment by richardshiue** [comment]
Hi @AmirRezaM75, this issue should be resolved in version 0.3.0. Could you try the latest version and let me know if you still experience this issue?

**Comment by AmirRezaM75** [comment]
@richardshiue @annieappflowy 
Thanks, it works in version 0.3.0
